content/ecosystem/clients/element-x.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Element X"
+template = "ecosystem/client.html"
+[extra]
+thumbnail = "element-x.svg"
+maintainer = "Element"
+maturity = "Beta"
+repo = "https://github.com/vector-im"
+matrix_room = "#element-x-ios:matrix.org"
```

Comment:
Feels like an oversight of the data model that we only support one room when there are 2 rooms 😬 (iirc there is also an android room for element x specifically)

Reply:
Hm yes it's worth fixing it. We should definitely support the room variants. I wonder if we should change the data model more in-depth too.

I quite like having one tile per "project", even if it spans across several platforms

Reply:
I guess this kinda is also related to https://github.com/matrix-org/matrix.org/issues/2010